resolver/dns_resolver.ml

```diff
@@ -299,12 +329,13 @@ let handle_primary t now ts proto sender sport packet _request buf =
             (* After guessing that a domain is blocked we add [`Filtered]
                extended error code and emit a [`Blocked] metrics event. *)
             let edns =
+              let reason = blocking_reason reply in
```

Comment:
We could add a few more words. For the dnsvizor PR it would only contain the domain nameified URL of the block list. Then again it's hard to know what people might put in the SOA hostmaster in the future.

Reply:
see [babc797](https://github.com/mirage/ocaml-dns/pull/384/commits/babc7974ab98adade32d9d844785a5b5f306211b)